table_test.go

```diff
@@ -114,6 +116,52 @@ func TestCreatePresplitTable(t *testing.T) {
 	}
 }
 
+func TestCreateTableWithAttributes(t *testing.T) {
+	testTableName := t.Name() + "_" + getTimestampString()
+	t.Log("testTableName=" + testTableName)
+
+	attrs := map[string]string{
+		"NORMALIZATION_ENABLED": "TRUE",
+	}
+
+	ac := gohbase.NewAdminClient(*host)
+	crt := hrpc.NewCreateTable(
+		context.Background(),
+		[]byte(testTableName),
+		cFamilies,
+		hrpc.TableAttributes(attrs),
+	)
+
+	if err := ac.CreateTable(crt); err != nil {
+		t.Errorf("CreateTable returned an error: %v", err)
+	}
+
+	// check in hbase:meta if there's a region for the table
+	c := gohbase.NewClient(*host)
+	metaKey := testTableName + ","
+	keyFilter := filter.NewPrefixFilter([]byte(metaKey))
+	scan, err := hrpc.NewScanStr(context.Background(), metaTableName, hrpc.Filters(keyFilter))
+	if err != nil {
+		t.Fatalf("Failed to create Scan request: %s", err)
+	}
+
+	var rsp []*hrpc.Result
+	scanner := c.Scan(scan)
+	for {
+		res, err := scanner.Next()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			t.Fatal(err)
+		}
+		rsp = append(rsp, res)
+	}
+	if len(rsp) != 1 {
+		t.Errorf("Meta returned %d rows for prefix '%s' , want 1", len(rsp), metaKey)
```

Comment:
This is checking that the table was created successfully. Are we able to check that the desired attribute is set?

Reply:
Right, the test isn’t great. Gohbase itself doesn’t have a method to get the attribute back afaik. Could potentially be added

Reply:
Ack